infrastructure/resource_group_level/main.bicep

```diff
@@ -1,4 +1,5 @@
 param location string  = resourceGroup().location
+// Expected values are 'feature', 'dev', 'test', 'staging' & 'prod'
```

Comment:
I think there is confusion between `qa`, `test` & `staging`. I think they are all the same, right?

Reply:
`staging` and `test` are the same. - they got merged into `staging` in the end